src/lib/pipeline.ml

```diff
@@ -168,6 +168,20 @@ module Full (Bfx: Extended_edsl.Semantics) = struct
       stringtie = bam |> Bfx.stringtie |> Bfx.save "stringtie";
       rna_bam_flagstat = bam |> Bfx.flagstat |> Bfx.save "rna-bam-flagstat";}
 
+  let rna_kallisto ~reference_build ?rna_fastqs () =
+    match rna_fastqs with
+    | None -> None
+    | Some rfqs ->
+      Some
+        (List.mapi
+          ~f:(fun i fq ->
+            let name = sprintf "Kallisto:rna-%d" i in
```

Comment:
@ihodes: this is now done. Had to refactor a few things, but I think the pipeline module now looks much better ;)